### ISSUE TITLE
ncurses  to 6.3

### ISSUE THREAD

**Issue by nobodyatandnothing** [clearlinux/distribution#2569]

**Comment by phmccarty** [comment]
Thanks for the notice!

I tried updating to 6.3 a couple weeks ago without success, and didn't follow up on that work yet... The packaging fails because pkgconfig files are not installed anymore:

```
...
Processing files: ncurses-dev-6.3-64.x86_64
error: File not found: /builddir/build/BUILDROOT/ncurses-6.3-64.x86_64/usr/lib64/pkgconfig/*.pc
```

**Comment by phmccarty** [comment]
The pkgconfig file configuration was revised in ncurses 6.3, and these changes had the side effect of not installing the .pc files.

I patched the `configure` script in ncurses 6.3 to avoid the problem for now.

The update should land in the next release.

**Comment by phmccarty** [comment]
Updated in release [35950](https://cdn.download.clearlinux.org/releases/35950/clear/RELEASENOTES).
